src/includes/errors.rs

```diff
@@ -1,5 +1,4 @@
-#[derive(Clone, Copy, PartialEq, Eq)]
-#[cfg_attr(any(feature = "debug", feature = "std"), derive(Debug))]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```

Comment:
is this file generated? it now has to be to support the different settings

Reply:
It's not generated. I moved all the stuff that needs to be generated out of it, except for derive(Debug).

Figured it makes sense for an error type to always derive Debug